indico/modules/rb/client/js/common/rooms/edit/RoomEditNotifications.jsx

```diff
@@ -13,7 +13,7 @@ import {FinalEmailList} from 'indico/react/components';
 import {FieldCondition, FinalCheckbox, FinalInput, validators as v} from 'indico/react/forms';
 import {Translate} from 'indico/react/i18n';
 
-export default function RoomEditNotifications({active}) {
+export default function RoomEditNotifications({active, roomNotificationSettings}) {
```

Comment:
I'd rename `roomNotificationSettings` to something like `defaults` or `globals` or `globalValues`